release/models/mpls/openconfig-mpls-ldp.yang

```diff
@@ -918,6 +942,32 @@ module openconfig-mpls-ldp {
     }
   }
 
+  grouping mpls-ldp-label-advertisement-config {
+    description
+      "Grouping containing configuration to enable Downstream on demand in a session.";
+    leaf downstream-on-deman-enabled {
+      type boolean;
+      default "false";
+      description
+        "'true' if session downstream on demand is enabled.";
+    }
+  }
+
+  grouping mpls-ldp-label-advertisement-state {
+    description
+      "Grouping containing Label advertisement mode state based on peer negotiaton.";
+    leaf peer-label-advertisement-mode {
+      type label-adverstisement-mode;
```

Comment:
`type` name will need updating after the `typedef` name typo is fixed :-)

Reply:
fixed typos, should be ok now